src/ontology/mondo-edit.obo

```diff
@@ -17,12 +17,14 @@ subsetdef: ordo_clinical_situation "particular clinical situation in a disease o
 subsetdef: ordo_clinical_subtype "clinical subtype"
 subsetdef: ordo_clinical_syndrome "clinical syndrome"
 subsetdef: ordo_disease "disease"
+subsetdef: ordo_disorder "disorder"
```

Comment:
@matentzn I added these subsetdefs, does this look okay?

Reply:
yes!